utils/sb2-config-gcc-toolchain

```diff
@@ -226,8 +226,8 @@ if [ $? != 0 ]; then
 	exit 1
 fi
 
-SBOX_CROSS_GCC_DIR=$(dirname $(which $GCC))
-if [ $GCC -v > /dev/null 2>&1 != 0 ]; then
+SBOX_CROSS_GCC_DIR="$(readlink -f $(dirname $(which $GCC)))"
```

Comment:
BTW, wouldn't it be better fixed on sdk-setup side? The double slash seems to come from there on sb2-init command line.

Reply:
I think so too.